packages/core/src/types.ts

```diff
@@ -525,6 +525,7 @@ export type GeneratorVerbOptions = {
   formUrlEncoded?: GeneratorMutator;
   override: NormalizedOverrideOutput;
   deprecated?: boolean;
+  originalSchema: OperationObject;
```

Comment:
originalSchema already exist in the code. What do you think about originalOperation?

Reply:
sure sounds good, updated.